lib/fragmentSender.js

```diff
@@ -80,6 +81,13 @@ module.exports = function(chat, fragments, payload, initialMessage = "", initial
         })
     } else {
         send_fragment(chat, messages, quick_replies);
+
+        const sendSubs = () => {
+            if (payload.slug == 'onboarding') {
+                subscriptions(chat);
+            }
+        }
+        sendSubs();
```

Comment:
1. Not sure what wrapping this code in an extra function archives. 
2. We do need to wait for `send_fragment` to finish before sending the subscriptions (implement `send_fragment` as a Promise) or else we will have random order. 
3. Won't this break when the onboarding story is expanded by a fragment?